src/createIsolation.tsx

```diff
@@ -36,29 +37,26 @@ export function createIsolation() {
     );
   };
 
-  const useAtom = ((anAtom: any, options?: any) => {
+  const useStore = ((options?: any) => {
     const store = useContext(StoreContext);
-    if (!store) {
-      throw new Error('Missing Provider from createIsolation');
-    }
+    if (!store) throw new Error('Missing Provider from createIsolation');
+    return options?.store || store;
```

Comment:
Thanks.
How about this for consistency?
```suggestion
    return useStoreOrig({ store, ...options });
```

Reply:
Thank you. I can't believe I missed that 😄